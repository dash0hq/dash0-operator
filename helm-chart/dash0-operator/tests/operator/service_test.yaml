suite: test role
templates:
  - operator/service.yaml
tests:
  - it: service should match snapshot (default settings)
    asserts:
      - matchSnapshot: {}

  - it: should render service with custom settings
    set:
      operator:
        metricsPort: 9554
    asserts:
      - equal:
          path: spec.ports[0].port
          value: 9554