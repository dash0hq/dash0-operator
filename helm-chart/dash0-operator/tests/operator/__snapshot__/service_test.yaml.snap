service should match snapshot (default settings):
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      labels:
        app.kubernetes.io/component: kube-rbac-proxy
        app.kubernetes.io/instance: controller-manager-metrics-service
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: service
        app.kubernetes.io/part-of: dash0-operator
        app.kubernetes.io/version: 1.0.0
        control-plane: controller-manager
        helm.sh/chart: dash0-operator-1.0.0
      name: dash0-operator-controller-manager-metrics-service
      namespace: NAMESPACE
    spec:
      ports:
        - name: https
          port: 8443
          protocol: TCP
          targetPort: https
      selector:
        control-plane: controller-manager
